src/pheval/post_processing/post_processing.py

```diff
@@ -44,7 +66,15 @@ def from_gene_result(pheval_gene_result: PhEvalGeneResult, rank: int):
 
 @dataclass
 class PhEvalVariantResult(PhEvalResult):
-    """Minimal data required from tool-specific output for variant prioritisation."""
+    """Minimal data required from tool-specific output for variant prioritisation
+    Args:
+        chromosome (str): The chromosome position of the variant
```

Comment:
Does it matter if the chromosome is in UCSC e.g. `chr1` or INSDC format e.g. `1` what about sex and mitochondrial chromosomes?  Should these be numeric or letter forms e.g. `23`, `24`, `25` or `X`, `Y`, `M` and is `M` or `MT` required for mitochondrial? How will this affect the results if these are incorrect?